### PR TITLE
DEVX-53: Updated documentation on logger migration.

### PR DESCRIPTION
**Pull request by kostya-luxuryescape**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,98 @@ See [Creating Custom Formats](https://www.npmjs.com/package/winston#creating-cus
 The contextual logger uses Node's `async_hooks` functionality to add Request IDs to the log output.
 
 ```ts
-import { createContextualLogger } from '@luxuryescapes/lib-logger';
+const {
+  createContextualLogger
+} = require("@luxuryescapes/lib-logger/lib/requestAwareLogger");
```
Comment (keithbro) [inline README.md:63]: I don't love digging in to an external lib like this. To me, any exposes functions should be at the root level.

Also do we want to have a require example and an import example. `import` is the more modern way so feels wrong to document only the legacy approach .

File: README.md [hunk 2]
```diff
@@ -58,7 +58,98 @@ See [Creating Custom Formats](https://www.npmjs.com/package/winston#creating-cus
...
+    ```js
+      "rules": {
+        "prettier/prettier": "error",
+        "no-undef": "error",
+        "no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
+        "no-unused-expressions": "error",
+        "no-console": "error" // <===================
+      },
+    ```
+* You can still live some console logs in scripts are not part of production code.
```
Comment (keithbro) [inline README.md:155]: s/live/leave/

**Review by keithbro — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (kostya-luxuryescape) [reply README.md:155]: wanted to finish typing before showcase while I couldn't spell on a good day :)

File: README.md (hunk 1 above)
Comment (kostya-luxuryescape) [reply README.md:63]: hmm, I'm actually not sure why I changed that 🤷‍♂️ 

really good point.

**Pull request merged**